src/Pages/TrendingPage/TrendingPage.jsx

```diff
@@ -0,0 +1,26 @@
+import { Loader, VideoCard } from "../../Components";
+import { useReducerContext } from "../../Context";
+import { v4 as uuid } from "uuid";
+import "./TrendingPage.css";
+
+export const TrendingPage = () => {
+  const { videos, loading } = useReducerContext();
+  return (
+    <section>
+      <div className="trending-header">
+        <h1>Trending Today</h1>
+      </div>
+      {loading ? (
+        <Loader />
+      ) : (
+        <div className="trending-video-container">
+          {videos
+            .filter((video) => (video.trending ? video : null))
```

Comment:
In the filter function you can just give a condition.. it'll automatically return the looped element if it's true, else null.
like: `videos.filter(video => video.trending)`

Reply:
thanks for pointing it out i will change it